com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/Magnifier.shader

```diff
@@ -0,0 +1,96 @@
+Shader "Graphics Tools/Magnifier"
+{
```

Comment:
Can you move this shader into the Experimental>Magnifier>Shaders folder? That way all magnifier assets and contained together within the same folder (the Acrylic folder is a good example to base off of).